nengo_viz/components/netgraph.py

```diff
@@ -143,7 +177,7 @@ def act_feedforward_layout(self, uid):
                                         pos=self.config[obj].pos,
                                         size=self.config[obj].size))
         self.config[network].has_layout = True
-        self.viz.viz.save_config()
+        self.viz.viz.save_config()"""
```

Comment:
Yes, this code which is commented out in netgraph.py should be removed. We just kept it there during development for experimenting with different designs. 

Most of this functionality has been moved to action.py.